content/S2016/S16-event-machine-learning.md

```diff
@@ -25,13 +25,13 @@ Technical Workshop Series: This event is generously sponsored by
 Technical Workshop Series. We are hosting three such workshops 
 focused on teaching technical skills in partnership with BlackBerry. 
 To learn more about our Technical Workshop Series and partnership 
-with BlackBerry see our [sponsors page](http://wics.uwaterloo.ca/sponsorship/).
+with BlackBerry see our [sponsors page]({filename}/pages/sponsorship.md).
```

Comment:
woohoo, thanks for checking it too

can you go through the rest of the links to double check they work?